adafruit_lc709203f.py

```diff
@@ -113,102 +121,102 @@ class LC709203F:
+    def cell_temperature(self) -> float:
         """Returns the temperature of the cell"""
         return self._read_word(LC709203F_CMD_CELLTEMPERATURE) / 10 - 273.15
 
     @cell_temperature.setter
-    def cell_temperature(self, value):
+    def cell_temperature(self, value: float) -> None:
         """Sets the temperature in the LC709203F"""
         if self.thermistor_enable:
             raise AttributeError("temperature can only be set in i2c mode")
         self._write_word(LC709203F_CMD_CELLTEMPERATURE, int(value + 273.15) * 10)
 
     @property
-    def ic_version(self):
+    def ic_version(self) -> int:
         """Returns read-only chip version"""
         return self._read_word(LC709203F_CMD_ICVERSION)
 
     @property
-    def power_mode(self):
+    def power_mode(self) -> Literal[0, 1]:
         """Returns current power mode (operating or sleeping)"""
         return self._read_word(LC709203F_CMD_POWERMODE)
 
     @power_mode.setter
-    def power_mode(self, mode):
+    def power_mode(self, mode: Literal[0, 1]) -> None:
         if not PowerMode.is_valid(mode):
             raise AttributeError("power_mode must be a PowerMode")
         self._write_word(LC709203F_CMD_POWERMODE, mode)
 
     @property
-    def battery_profile(self):
+    def battery_profile(self) -> Literal[0, 1]:
         """Returns current battery profile (0 or 1)"""
         return self._read_word(LC709203F_CMD_BATTPROF)
 
     @battery_profile.setter
-    def battery_profile(self, mode):
+    def battery_profile(self, mode: Literal[0, 1]) -> None:
         if not mode in (0, 1):
             raise AttributeError("battery_profile must be 0 or 1")
         self._write_word(LC709203F_CMD_BATTPROF, mode)
 
     @property
-    def pack_size(self):
+    def pack_size(self) -> int:
         """Returns current battery pack size"""
         return self._read_word(LC709203F_CMD_APA)
 
     @pack_size.setter
-    def pack_size(self, size):
+    def pack_size(self, size: int) -> None:
         if not PackSize.is_valid(size):
             raise AttributeError("pack_size must be a PackSize")
         self._write_word(LC709203F_CMD_APA, size)
 
     @property
-    def thermistor_bconstant(self):
+    def thermistor_bconstant(self) -> int:
         """Returns the thermistor B-constant"""
         return self._read_word(LC709203F_CMD_THERMISTORB)
 
     @thermistor_bconstant.setter
-    def thermistor_bconstant(self, bconstant):
+    def thermistor_bconstant(self, bconstant: int) -> None:
         """Sets the thermistor B-constant"""
         self._write_word(LC709203F_CMD_THERMISTORB, bconstant)
 
     @property
-    def thermistor_enable(self):
+    def thermistor_enable(self) -> int:
         """Returns the current temperature source"""
         return self._read_word(LC709203F_CMD_STATUSBIT)
 
     @thermistor_enable.setter
-    def thermistor_enable(self, status):
+    def thermistor_enable(self, status: Union[Literal[0, 1], bool]) -> None:
```

Comment:
`status` can be bool, which is the union of `True` and `False`.  A better guard statement, therefore, that can be used here if you want to change it is:

```python
if not isinstance(status, bool):
```

Reply:
but not the annotation in the def?

Reply:
In the documentation, only the getter's docstring is used.  It's true that it's not super clear what the property does, so we could change it to something like "The enable status of the thermistor as a temperature source".  Does that seem clear?